<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>interfaz1</class>
 <widget class="QMainWindow" name="interfaz1">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>991</width>
    <height>482</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Calculadora Ti:Zafiro</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="CW">
       <attribute name="title">
        <string>Onda continua</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="1" column="0" rowspan="3">
         <layout class="QFormLayout" name="formLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="longitudL1Label">
            <property name="text">
             <string>Longitud L1</string>
            </property>
            <property name="buddy">
             <cstring>L1SpinBox</cstring>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="longitudL2Label">
            <property name="text">
             <string>Longitud L2</string>
            </property>
            <property name="buddy">
             <cstring>L2SpinBox</cstring>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="longitudCristalLabel">
            <property name="text">
             <string>Longitud Cristal</string>
            </property>
            <property name="buddy">
             <cstring>LSpinBox</cstring>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="indiceDeRefraccionLabel">
            <property name="text">
             <string>Índice de refracción</string>
            </property>
            <property name="buddy">
             <cstring>nSpinBox</cstring>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="LambdaIntracavidadLabel">
            <property name="text">
             <string>λ intracavidad</string>
            </property>
            <property name="buddy">
             <cstring>lambdaLSpinBox</cstring>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QDoubleSpinBox" name="L1SpinBox">
            <property name="buttonSymbols">
             <enum>QAbstractSpinBox::NoButtons</enum>
            </property>
            <property name="suffix">
             <string>[m]</string>
            </property>
            <property name="maximum">
             <double>100000.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QDoubleSpinBox" name="L2SpinBox">
            <property name="buttonSymbols">
             <enum>QAbstractSpinBox::NoButtons</enum>
            </property>
            <property name="suffix">
             <string>[m]</string>
            </property>
            <property name="maximum">
             <double>100000.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QDoubleSpinBox" name="LSpinBox">
            <property name="buttonSymbols">
             <enum>QAbstractSpinBox::NoButtons</enum>
            </property>
            <property name="suffix">
             <string>[mm]</string>
            </property>
            <property name="maximum">
             <double>100000.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QDoubleSpinBox" name="nSpinBox">
            <property name="buttonSymbols">
             <enum>QAbstractSpinBox::NoButtons</enum>
            </property>
            <property name="decimals">
             <number>4</number>
            </property>
            <property name="maximum">
             <double>100000.000000000000000</double>
            </property>
            <property name="value">
             <double>0.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QDoubleSpinBox" name="lambdaLSpinBox">
            <property name="buttonSymbols">
             <enum>QAbstractSpinBox::NoButtons</enum>
            </property>
            <property name="suffix">
             <string>[nm]</string>
            </property>
            <property name="maximum">
             <double>100000.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="distanciaFocalM1Label">
            <property name="text">
             <string>Distancia focal M1</string>
            </property>
            <property name="buddy">
             <cstring>f1SpinBox</cstring>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QDoubleSpinBox" name="f1SpinBox">
            <property name="buttonSymbols">
             <enum>QAbstractSpinBox::NoButtons</enum>
            </property>
            <property name="suffix">
             <string>[mm]</string>
            </property>
            <property name="maximum">
             <double>1000000.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="6" column="0">
           <widget class="QLabel" name="distanciaFocalM2Label">
            <property name="text">
             <string>Distancia focal M2</string>
            </property>
            <property name="buddy">
             <cstring>f2SpinBox</cstring>
            </property>
           </widget>
          </item>
          <item row="6" column="1">
           <widget class="QDoubleSpinBox" name="f2SpinBox">
            <property name="buttonSymbols">
             <enum>QAbstractSpinBox::NoButtons</enum>
            </property>
            <property name="prefix">
             <string/>
            </property>
            <property name="suffix">
             <string>[mm]</string>
            </property>
            <property name="maximum">
             <double>1000000.000000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Conjugado en EM1</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QRadioButton" name="EM2Fin">
            <property name="text">
             <string>Finito</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">conjugadoL2</string>
            </attribute>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Conjugado en EM2</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QRadioButton" name="EM2Inf">
            <property name="text">
             <string>Infinito</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">conjugadoL2</string>
            </attribute>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QRadioButton" name="EM1Fin">
            <property name="text">
             <string>Finito</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">conjugadoL1</string>
            </attribute>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QRadioButton" name="EM1Inf">
            <property name="text">
             <string>Infinito</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">conjugadoL1</string>
            </attribute>
           </widget>
          </item>
         </layout>
        </item>
        <item row="2" column="1" colspan="2">
         <widget class="QPushButton" name="initButton">
          <property name="text">
           <string>Corre</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QGroupBox" name="estabilidadGroupBox">
          <property name="title">
           <string>Mapa de estabilidad</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_4">
           <item row="3" column="0">
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>ε2 mínimo</string>
             </property>
             <property name="buddy">
              <cstring>epsilon2_min</cstring>
             </property>
            </widget>
           </item>
           <item row="3" column="2">
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>ε2 máximo</string>
             </property>
             <property name="buddy">
              <cstring>epsilon2_max</cstring>
             </property>
            </widget>
           </item>
           <item row="3" column="3">
            <widget class="QDoubleSpinBox" name="epsilon2_max">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="suffix">
              <string>[mm]</string>
             </property>
             <property name="minimum">
              <double>-99.989999999999995</double>
             </property>
             <property name="maximum">
              <double>99.989999999999995</double>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>ε1 mínimo</string>
             </property>
             <property name="buddy">
              <cstring>epsilon1_min</cstring>
             </property>
            </widget>
           </item>
           <item row="2" column="5">
            <widget class="QDoubleSpinBox" name="pasoEpsilon1Box">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="suffix">
              <string>[mm]</string>
             </property>
             <property name="minimum">
              <double>-99.989999999999995</double>
             </property>
             <property name="maximum">
              <double>99.989999999999995</double>
             </property>
             <property name="singleStep">
              <double>0.100000000000000</double>
             </property>
             <property name="value">
              <double>0.000000000000000</double>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QDoubleSpinBox" name="epsilon2_min">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="suffix">
              <string>[mm]</string>
             </property>
             <property name="minimum">
              <double>-99.989999999999995</double>
             </property>
             <property name="maximum">
              <double>99.989999999999995</double>
             </property>
            </widget>
           </item>
           <item row="3" column="4">
            <widget class="QLabel" name="label_8">
             <property name="text">
              <string>cada [mm]</string>
             </property>
             <property name="buddy">
              <cstring>pasoEpsilon2Box</cstring>
             </property>
            </widget>
           </item>
           <item row="2" column="2">
            <widget class="QLabel" name="label_4">
             <property name="toolTip">
              <string>Si la diferencia entre máximos y mínimos de ε1 y ε2
es muy grande, el cálculo no lineal requiere más tiempo. 
Entre más pequeño sea el paso, más tiempo será necesario.</string>
             </property>
             <property name="text">
              <string>ε1 máximo</string>
             </property>
             <property name="buddy">
              <cstring>epsilon1_max</cstring>
             </property>
            </widget>
           </item>
           <item row="2" column="4">
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>cada [mm]</string>
             </property>
             <property name="buddy">
              <cstring>pasoEpsilon1Box</cstring>
             </property>
            </widget>
           </item>
           <item row="2" column="3">
            <widget class="QDoubleSpinBox" name="epsilon1_max">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="suffix">
              <string>[mm]</string>
             </property>
             <property name="minimum">
              <double>-99.989999999999995</double>
             </property>
             <property name="maximum">
              <double>99.989999999999995</double>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QDoubleSpinBox" name="epsilon1_min">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="suffix">
              <string>[mm]</string>
             </property>
             <property name="minimum">
              <double>-99.989999999999995</double>
             </property>
             <property name="maximum">
              <double>99.989999999999995</double>
             </property>
            </widget>
           </item>
           <item row="3" column="5">
            <widget class="QDoubleSpinBox" name="pasoEpsilon2Box">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="suffix">
              <string>[mm]</string>
             </property>
             <property name="minimum">
              <double>-99.989999999999995</double>
             </property>
             <property name="maximum">
              <double>99.989999999999995</double>
             </property>
             <property name="singleStep">
              <double>0.100000000000000</double>
             </property>
            </widget>
           </item>
           <item row="0" column="0" colspan="3">
            <widget class="QCheckBox" name="mapaCW">
             <property name="layoutDirection">
              <enum>Qt::RightToLeft</enum>
             </property>
             <property name="text">
              <string>¿Generar mapa de estabilidad?</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QPushButton" name="initCalculloCW">
          <property name="text">
           <string>Corre cálculo CW</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>Tab 2</string>
       </attribute>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>510</x>
          <y>290</y>
          <width>281</width>
          <height>116</height>
         </rect>
        </property>
        <layout class="QFormLayout" name="formLayout_2">
         <item row="1" column="1">
          <widget class="CtoCppQProgressBar" name="avanceTermico">
           <property name="value">
            <number>0</number>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>Progreso cálculo térmico</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>Cálculo spots acople</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="CtoCppQProgressBar" name="avanceSpots">
           <property name="value">
            <number>0</number>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_11">
           <property name="text">
            <string>Cálculo no lineal EM1</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_12">
           <property name="text">
            <string>Cálculo no lineal EM2</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="CtoCppQProgressBar" name="avanceNoLinealEM1">
           <property name="value">
            <number>0</number>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="CtoCppQProgressBar" name="avanceNoLinealEM2">
           <property name="value">
            <number>0</number>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>991</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuArchivo">
    <property name="title">
     <string>Archivo</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionImporta_constantes"/>
    <addaction name="actionOpcionesNoLineal"/>
    <addaction name="separator"/>
    <addaction name="actionSalir"/>
   </widget>
   <widget class="QMenu" name="menuAyuda">
    <property name="title">
     <string>Ayuda</string>
    </property>
    <addaction name="actionVer_diagrama"/>
    <addaction name="actionAcerca_de"/>
   </widget>
   <addaction name="menuArchivo"/>
   <addaction name="menuAyuda"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionImporta_constantes">
   <property name="text">
    <string>Importa constantes</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSalir">
   <property name="text">
    <string>Salir</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionOpcionesNoLineal">
   <property name="text">
    <string>Opciones de cálculo</string>
   </property>
   <property name="shortcut">
    <string>Alt+O</string>
   </property>
  </action>
  <action name="actionAcerca_de">
   <property name="text">
    <string>Acerca de...</string>
   </property>
  </action>
  <action name="actionVer_diagrama">
   <property name="text">
    <string>Ver diagrama</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CtoCppQProgressBar</class>
   <extends>QProgressBar</extends>
   <header>ctocppqprogressbar.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>EM1Fin</tabstop>
  <tabstop>EM1Inf</tabstop>
  <tabstop>EM2Fin</tabstop>
  <tabstop>EM2Inf</tabstop>
  <tabstop>L1SpinBox</tabstop>
  <tabstop>L2SpinBox</tabstop>
  <tabstop>LSpinBox</tabstop>
  <tabstop>nSpinBox</tabstop>
  <tabstop>lambdaLSpinBox</tabstop>
  <tabstop>f1SpinBox</tabstop>
  <tabstop>f2SpinBox</tabstop>
  <tabstop>initButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="conjugadoL2"/>
  <buttongroup name="conjugadoL1"/>
 </buttongroups>
</ui>
